lib/bitcoin/utxo.rb

```diff
@@ -0,0 +1,38 @@
+module Bitcoin
+  class Utxo
```

Comment:
I think this class is a wallet related class, so how about placing it under `Bitcoin::Wallet`?

Reply:
As you mentioned, it may be better to put it under bitcoin/wallet.

Reply:
Sure, I will modify the code to move Utxo class to under Bitcoin::Wallet from Bitcoin module.